api/RTCIceCandidatePairStats.json

```diff
@@ -0,0 +1,1635 @@
+{
+  "api": {
+    "RTCIceCandidatePairStats": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceCandidatePairStats",
+        "support": {
+          "chrome": {
+            "version_added": true
```

Comment:
Either [this interface was never implemented](https://cs.chromium.org/search/?q=RTCIceCandidatePairStats+file:%5Esrc/third_party/blink/renderer/+package:%5Echromium$&type=cs) or at some point the entire interface was removed from Chrome, Android, and Webview. I cannot tell you when.

Reply:
Well, it's a dictionary, so it wouldn't be something you create with `stats = new RTCIceCandidatePairStats()` or something.  It's a component of the main `RTCStats` object. Looking through the code, I see that the data is collected and the object is available.

See:
* https://cs.chromium.org/chromium/src/third_party/webrtc/stats/rtcstats_objects.cc?q=rtcicecandidatepairstats&sq=package:chromium&l=183&dr=CSs
* https://cs.chromium.org/chromium/src/content/renderer/media/webrtc/rtc_stats.cc?q=rtcicecandidatepairstats&sq=package:chromium&dr=C&l=27
* https://cs.chromium.org/chromium/src/third_party/webrtc/pc/rtc_stats_collector.h?q=rtcicecandidatepairstats&sq=package:chromium&l=172&dr=C
* https://cs.chromium.org/chromium/src/third_party/webrtc/pc/rtc_stats_collector.cc?q=rtcicecandidatepairstats&sq=package:chromium&l=1093&dr=C

I was not able to discern when i twas added within a reasonable time period.

Reply:
It was [committed here](https://webrtc.googlesource.com/src.git/+/c47a0c3ac42ba4a5582187acc3ad3a762c323e33). That looks like Chrome 56.

Reply:
Changed for the overall dictionary. Not going to try to deal with the details further right now.